executor/oci/user.go

```diff
@@ -84,3 +94,24 @@ func parseUID(str string) (uint32, error) {
 	}
 	return uint32(uid), nil
 }
+
+// FIXME: This is a temporeray fix for the missing supplementary GIDs from containerd
```

Comment:
nit: reorg these comments so they start with function name like is required for godoc.